setup.py

```diff
@@ -142,11 +156,20 @@ def collect_external_package_data(external_package_data):
     library_dirs += opencv_library_dirs
     libraries += opencv_libraries
 
-    # Eigen
-    include_dirs += [
-        "/usr/local/include/eigen3",
-        "/usr/include/eigen3",
-    ]
+    if is_conda:
```

Comment:
This variable won't be defined in case there is no conda environment and a NameError will be raised. I suggest using the same check as above.
```suggestion
    if 'conda' in os.__file__:
```

Reply:
@papr works as expected on my Windows machine – installed to fresh conda env